packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -37,18 +37,19 @@ import type { TableHandlesProsemirrorPlugin } from "../extensions/TableHandles/T
 import { UniqueID } from "../extensions/UniqueID/UniqueID.js";
 import type { Dictionary } from "../i18n/dictionary.js";
 import { en } from "../i18n/locales/index.js";
-import type {
-  BlockIdentifier,
-  BlockNoteDOMAttributes,
-  BlockSchema,
-  BlockSpecs,
-  CustomBlockNoteSchema,
-  InlineContentSchema,
-  InlineContentSpecs,
-  PartialInlineContent,
-  Styles,
-  StyleSchema,
-  StyleSpecs,
+import {
+  partialBlockToFullBlock,
+  type BlockIdentifier,
```

Comment:
fyi, some of the changes in `type` imports were to fix a circular dependency issue that was breaking the build during development